backend/api/Controllers/ScheduledMissionController.cs

```diff
@@ -0,0 +1,106 @@
+﻿using Api.Controllers.Models;
+using Api.Models;
+using Api.Services;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Api.Controllers;
+
+[ApiController]
+[Route("scheduledMission")]
```

Comment:
```suggestion
[Route("scheduled-missions")]
```
REST naming conventions